package/gluon-mesh-batman-adv-brmldproxy/files/usr/sbin/gluon-brmldproxy-router-check

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+lookup_site() {
+	local path="$1" default="$2"
+	lua -e "print(require('gluon.site').$path('$default'))"
+}
+
+lookup_prefix6() {
+	local prefix
+
+	prefix="$(lookup_site 'prefix6')"
+	echo "${prefix%/*}"
+}
+
+lookup_prefix6_len() {
+	local prefix
+
+	prefix="$(lookup_site 'prefix6')"
+	echo "${prefix#*/}"
+}
+
+# Generates and prints an RFC3306, section 4 style network prefix based
+# multicast address from the site prefix6 with a pseudo-random group-id
+# from the domain seed.
+get_gluon_all_mc_routers_ip6() {
+	local prefix6 prefix6_len group_id
+
+	prefix6="$(lookup_prefix6)"
+	prefix6_len="$(lookup_prefix6_len)"
```

Comment:
Probably a good idea to avoid running `lookup_site` twice, and instead just assign the output of `lookup_site 'prefix6'` to `prefix6`, splitting it in the subsequent uses below.